### PR TITLE
Add Learn to Code - Go course

### PR DESCRIPTION
**Pull request by dlsniper**
Added introduction course to programming via Go.

/cc @carlisia @eggya

### CODE REVIEW

**Comment by dlsniper** [comment]
Once this PR is merged, we can announce [the event](https://www.bridgetroll.org/events/328) in London next year.

File: app/models/course.rb [hunk 1]
```diff
@@ -611,7 +611,52 @@ class Course < ActiveHash::Base
...
-    }
+    }, {
+      id: 17,
+      name: 'Learn to Code - Go',
+      title: 'Learn Code with Go',
+      description: 'This is an introductory event. The focus will be on learning programming fundamentals using Go.',
+      levels: [
+        {
+          level: 1,
+          color: 'green',
```
Comment (lilliealbert) [inline app/models/course.rb]: @dlsniper It probably doesn't make sense to have two levels with the same color. Can you change this one to blue?

Comment (dlsniper) [inline app/models/course.rb]: Done, good catch.

**Comment by carlisia** [comment]
I wish a little bit that the description could be more extense. Other than that, 👍

**Pull request merged**